ncm-network/src/main/pan/components/network/types/network/backend/initscripts.pan

```diff
@@ -1,3 +1,59 @@
 declaration template components/network/types/network/backend/initscripts;
 
 @{implement types specific for initscripts / network.pm}
+
+include 'components/network/types/network/route';
+
+type structure_network_backend_specific = {
+};
+
+@documentation{
+    Add route (IPv4 of IPv6)
+    Presence of ':' in any of the values indicates this is IPv6 related.
+}
+type network_route = {
+    @{The ADDRESS in ADDRESS/PREFIX via GATEWAY}
```

Comment:
i would factor out all common bits. i don't know why you have this in the backend code

Reply:
This seemed the easiest option for having different validation rules per back-end, but I now realise I could redefine the validation function in each instead.

Reply:
I'll try and refactor this tomorrow.